src/Behavioral.Automation/Bindings/TableBinding.cs

```diff
@@ -1,105 +1,137 @@
-
-        [Given("(.*?) has no records")]
-        [Then("(.*?) should contain no records")]
-        public void CheckSearchResultIsEmpty(ITableWrapper table)
-        {
-            Assert.ShouldBecome(() => table.Displayed && !table.Rows.Any(), true, $"{table.Caption} contains records");
-        }
-
-        [Given("(.*?) element among (.*) (is| is not) expanded")]
-        [When("(.*?) element among (.*) (is| is not) expanded")]
-        [Then("(.*?) element among (.*) should (be|be not) expanded")]
-        public void CheckRowIsExpanded(int index, IElementCollectionWrapper table, string behavior)
-        {
-            var check = table.GetAttributeByIndex("class", index-1).Contains("expandable-expanded-row");
-            check.Should().Be(!behavior.Contains("not"));
-        }
-    }
-}
+            else
+            {
+                Assert.ShouldBecome(() => gridRows.CellsText.Count() == expectedValues.Count, true,
+                    $"{gridRows.Caption} is {gridRows.CellsText.Aggregate((x, y) => $"{x}, {y}")}");
+
+                Assert.ShouldBecome(() => gridRows.CellsText.HaveValues(expectedValues, false),
+                    true,
+                    $"{gridRows.Caption} is {gridRows.CellsText.Aggregate((x, y) => $"{x}, {y}")}");
+            }
+        }
+
+        [Then("(.+?) should have in exact order the following rows:")]
+        public void CheckTableHaveRowsInExactOrder(ITableWrapper gridRows, Table table)
+        {
+            var expectedValues = ListServices.TableToCellsList(table);
+
+            Assert.ShouldBecome(() => gridRows.Stale, false, $"{gridRows.Caption} is stale");
+
+            Assert.ShouldBecome(() => gridRows.CellsText.HaveValues(expectedValues, true),
+                true,
+                $"{gridRows.Caption} is {gridRows.CellsText.Aggregate((x, y) => $"{x}, {y}")}");
+        }
+
+        [Given("(.+?) (contains|does not contain) the following rows:")]
+        [Then("(.+?) should (contain|not contain) the following rows:")]
+        public void CheckTableContainsRows(ITableWrapper gridRows, string behavior,  Table table)
+        {
+            var expectedValues = ListServices.TableToCellsList(table);
+
+            Assert.ShouldBecome(() => gridRows.Stale, false, $"{gridRows.Caption} is stale");
+
+            bool inversion = behavior.Contains("not");
+            if (inversion)
+            {
+                Assert.ShouldBecome(() => gridRows.CellsText.DoesntContainValues(expectedValues),
+                    true,
+                    $"{gridRows.Caption} is {gridRows.CellsText.Aggregate((x, y) => $"{x}, {y}")}");
+            }
+            else
+            {
+
+                Assert.ShouldBecome(() => gridRows.CellsText.ContainsValues(expectedValues),
+                    true,
+                    $"{gridRows.Caption} is {gridRows.CellsText.Aggregate((x, y) => $"{x}, {y}")}");
+            }
+        }
+
+        [Then("(.*?) should (have|not have) \"(.*)\" in (.*)")]

```

Comment:
looks like no example for this binding in BINDINGS.md

Reply:
added description and example